exercises/practice/anagram/.meta/tests.toml

```diff
@@ -40,6 +52,24 @@ description = "anagrams must use all letters exactly once"
 
 [85757361-4535-45fd-ac0e-3810d40debc1]
 description = "words are not anagrams of themselves (case-insensitive)"
+include = false
+
+[68934ed0-010b-4ef9-857a-20c9012d1ebf]
+description = "words are not anagrams of themselves"
+reimplements = "85757361-4535-45fd-ac0e-3810d40debc1"
+
+[589384f3-4c8a-4e7d-9edc-51c3e5f0c90e]
+description = "words are not anagrams of themselves even if letter case is partially different"
+reimplements = "85757361-4535-45fd-ac0e-3810d40debc1"
+
+[ba53e423-7e02-41ee-9ae2-71f91e6d18e6]
+description = "words are not anagrams of themselves even if letter case is completely different"
+reimplements = "85757361-4535-45fd-ac0e-3810d40debc1"
```

Comment:
Is this correct? 3 test cases `reimplement` the same test case

Reply:
Yes, this is correct.
The TL;DR is that the original test was split into three tests for clarity.

See https://github.com/exercism/problem-specifications/pull/1943 for the full explanation.

"Reimplements" means that the new test specification supersedes the old one. Since test specifications are (largely) immutable, if you want to make a material change to a test case, you essentially need to deprecate the old one and implement a new one. "Reimplements" is the mechanism for this.